lecsum-fe/pages/retry_exam.py

```
@@ -0,0 +1,474 @@
+        else:
+            st.error(f"재시험 생성 실패: {response.status_code} - {response.text}")
+            return None
+    except Exception as e:
+        st.error(f"서버 연결 실패: {e}")
+        return None
+
+
+def submit_retry_quiz(retry_quiz_set_id, quiz_id_list, user_answer_list):
+    """재시험 채점 API 호출"""
+    try:
+        response = requests.post(
```

Ruff (0.14.8)

Probable use of `requests` call without timeout

(S113)